platforms/hyperledger-besu/configuration/roles/create/certificates/ambassador/tasks/nested_main.yaml

```diff
@@ -102,14 +102,40 @@
     domain_name: "{{ node_name }}.{{ organizationItem.external_url_suffix }}"
   when: root_certs.failed == True
 
+# Generate crypto for besu nodes and knownserver file
+- name: create tls credentials for besu
+  shell: |
+    cd {{ ambassadortls }}
+    eval "openssl genrsa -out {{ node_name }}-besu-node.key 2048"
+    eval "openssl req -new -key {{ node_name }}-besu-node.key -out {{ node_name }}-besu-node.csr -subj '/CN={{ common_name }}'"
+    eval "openssl x509 -req -in {{ node_name }}-besu-node.csr -CA {{ rootca }}/rootca.pem -CAkey {{ rootca }}/rootca.key -CAcreateserial -out {{ node_name }}-besu-node.pem -days 500 -sha256"
+    eval "cat {{ node_name }}-besu-node.key {{ node_name }}-besu-node.pem > {{ node_name }}_certificates.pem"
+    eval "openssl pkcs12 -export -in {{ node_name }}_certificates.pem -out {{ node_name }}-besu-node.pkcs12 -password pass:somepass -name myAlias -noiter -nomaciter"
+    eval "echo "changeme" > {{ node_name }}-password"
+    eval "openssl x509 -in {{node_name}}-certchain.pem -noout -fingerprint -sha256 > {{node_name}}-certchain-sha256"
+    eval "export DIGEST=$(awk '{print $2}' {{node_name}}-certchain-sha256 | sed -n 's/Fingerprint=\(.*\)/\1/p')"
+    eval "echo "{{ common_name }} $DIGEST" > {{ node_name }}-knownServer"
```

Comment:
Don't we have to also add the k8s internal address in the knownserver as well?

Reply:
added the hostname and the port in the knownserver file and  hostname in deployment for node_orion